src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,7 +1150,7 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
+    private static String removePrefix(String s, String sign) {
```

Comment:
There's a lot more you need to do here.  Firstly, you mention prefix in the method head, but this is not apparent in the code (still uses sign).  Secondly, you need to now specifically replace prefix that only occurs at the start of fullString.